karma.conf.js

```diff
@@ -272,7 +272,7 @@ module.exports = (config) => {
               // Don't instrument these parts of the codebase.
               exclude: [
                 'demo/**/*.js',
-                'lib/(debug|deprecate|polyfill)/*.js',
+                'lib/(debug|deprecate|polyfill|lcevc)/*.js',
```

Comment:
Why exclude the lcevc parts of the library from code coverage stats?

Reply:
Entry for lcevc is  now removed.